scripts/databricks/add_monte_carlo_webhook_notifications.py

```diff
@@ -0,0 +1,69 @@
+from typing import Set
+
+import click
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.jobs import Webhook, WebhookNotifications
+
+
+@click.command(help="Add the monte carlo webhook to Databricks jobs")
+@click.pass_obj
+@click.option(
+    "--mc-notification-id",
+    required=True,
+    help="UUID of the existing Databricks Notification pointing to the MC Webhook endpoint",
+)
+@click.option(
+    "--job-name",
+    required=False,
+    multiple=True,
+    help="Job Name to add the MC Webhook to. Can be used multiple times. If not specified, add the MC Webhook to all jobs.",
+)
+def add_monte_carlo_webhook_notifications(ctx, **kwargs):
+    job_names = set(kwargs["job_name"])
+    _add_monte_carlo_webhook_notifications(
+        mc_notification_id=kwargs["mc_notification_id"], job_names=job_names
+    )
+
+
+def _add_monte_carlo_webhook_notifications(
+    mc_notification_id: str, job_names: Set[str]
+):
+    w = WorkspaceClient()
+
+    # assert that the notification exists
+    w.notification_destinations.get(id=mc_notification_id)
+    mc_notification = Webhook(id=mc_notification_id)
+
+    for job in w.jobs.list():
```

Comment:
Should we include some error handling + logging if no jobs were found and/or that the process is starting?